### PR TITLE
Merge Dark Sky docs

### PR DESCRIPTION
**Pull request by NeLLyMerC**
Attempting to merge Dark Sky documentation for simplification and clarity. If the PR is accepted, I'd like to remove /integrations/weather.darksky/ and just keep the current page (/integrations/darksky/)

## Type of change

- [x ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:
- This PR fixes or closes issue:

## Checklist

- [ x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
We would prefer that you now immediately remove the old platform page and set the redirects correctly from weather.darksky to darksky.

File: source/_integrations/darksky.markdown [hunk 1]
```diff
@@ -235,6 +237,48 @@ All language options are described in this table that you can use for the dark s
...
+  required: false
+  type: string
+  default: Dark Sky
+mode:
+  description: "The forecast type. Can be `hourly` or `daily`."
+  required: false
+  type: string
+  default: hourly
+{% endconfiguration %}
+
```
Comment (klaasnicolaas) [inline source/_integrations/darksky.markdown:286]: Remove the extra whitespace

File: source/_integrations/darksky.markdown [hunk 2]
```diff
@@ -235,6 +237,48 @@ All language options are described in this table that you can use for the dark s
 
 <div class='note warning'>
 While the platform is called "darksky" the sensors will show up in Home Assistant as "dark_sky" (eg: sensor.dark_sky_summary).
-</div>
+</div>  
+
```
Comment (klaasnicolaas) [inline source/_integrations/darksky.markdown:243]: Maybe add here a header with **Weather**

File: source/_integrations/darksky.markdown [hunk 3]
```diff
@@ -24,7 +24,9 @@ You need an API key which is free but requires [registration](https://darksky.ne
 
 ## Configuration
 
-To add Dark Sky to your installation, add the following to your `configuration.yaml` file:
+There are two ways you can integrate Dark Sky into Home Assistant: as a sensor and a weather entity. The [sensor](/integrations/sensor) entity allows you to add the conditions you'd like to monitor and returns a value that you can add to various styles of Lovelace cards. In contrast, the weather entity allows you to set up a [weather forecast card](/lovelace/weather-forecast/#configuration-variables) that displays a 5-day forecast with some additional data. 
+
```
Comment (klaasnicolaas) [inline source/_integrations/darksky.markdown:28]: Maybe add here a header with **Sensor**

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/darksky.markdown (hunk 3 above)
Comment (NeLLyMerC) [reply source/_integrations/darksky.markdown:28]: Added, thank you for the feedback

File: source/_integrations/darksky.markdown (hunk 1 above)
Comment (NeLLyMerC) [reply source/_integrations/darksky.markdown:286]: Whitespace removed, thank you.

**Review by NeLLyMerC — COMMENTED** [review]
Edited document based on feedback

**Comment by frenck** [comment]
These are 2 different things in Home Assistant. How would one know the difference know? Where is it explained? And how would that go with updates? 

I think we should keep things that are separate in our code base, separate in the documentation as well. Mainly since the documentation should reflect the codebase.

**Comment by NeLLyMerC** [comment]
> These are 2 different things in Home Assistant. How would one know the difference know? Where is it explained? And how would that go with updates?
> 
> I think we should keep things that are separate in our code base, separate in the documentation as well. Mainly since the documentation should reflect the codebase.

The differences are described in one of the first paragraphs on the edits I made. Each entity type is linked to a page for reference (/integration/sensors & /lovelace/weather-forecast/), so the user can see examples of the differences. There are also two separate headings with the example configuration needed to get each added to Home Assistant. I found it confusing as it is because searching “Dark Sky” in the integrations page yields two results that say the same thing. The current page of /integrations/weather.darksky is also not very descriptive of what you will gain by adding this type of integration.

**Comment by frenck** [comment]
I'm assigning myself as well, I think having weather platforms both implemented as weather and sensor actually are unfriendly. I'm going to take up this discussion on that level first.

**Comment by NeLLyMerC** [comment]
The idea I was going for is similar to other integrations like Canary, Bloomsky, or Arlo. One page for the integration that shows you the different entities that can be configured for it. 
Let me know how I can help

**Comment by frenck** [comment]
Yes, I get that, however, in this case it are 2 different integrations/platforms. (different source).

**Comment by frenck** [comment]
Not stale, thanks stable bot.

**Pull request closed without merging**